tests/semantic_analyser.cpp

```diff
@@ -1249,7 +1252,7 @@ TEST(semantic_analyser, call_str)
   test("kprobe:f { str(arg0); }");
   test("kprobe:f { @x = str(arg0); }");
   test("kprobe:f { str(); }", 1);
-  test("kprobe:f { str(\"hello\"); }", 1);
```

Comment:
I know it's not causing any issues to allow this but should we add a warning to tell the user this is unneccessary?

Reply:
It gets complicated. What if the user has `str("foo", 3)`? I'd rather just let it be a no-op than have complex logic around whether it is a warning or not (e.g. if the resulting fold is bounded or not).

Reply:
Yeah that's fair. Not a big deal.